coalib/io/FileFactory.py

```diff
@@ -70,6 +71,7 @@ def __init__(self, filename):
         """
         self._filename = os.path.abspath(filename)
         self._timestamp = os.path.getmtime(self._filename)
+        self.cache = {}
```

Comment:
is that necessary? I would expect from `memoized_property` to create the cache field itself^^